feat: Convert all images into 'O-portfolio' folder in .webp format. The result is into 'O-portfolio_webp' folder.

DevWeb13 opened the pull request:
Convert all images into 'O-portfolio' folder in .webp format. The result is into 'O-portfolio_webp' folder.

DevWeb13 committed:
feat: Convert all images into 'O-portfolio' folder in .webp format. The result is into 'O-portfolio_webp' folder.

IbraheemHaseeb7 commented:
did you @DevWeb13  just converted the pictures to webp format or also updated the pictures in the HTML?

DevWeb13 commented:
> did you @DevWeb13 just converted the pictures to webp format or also updated the pictures in the HTML?

Just converted the picture to webp format into the new folder: 'O-portfolio_webp'.
The original images are still in their folder: 'O-portfolio'.

If the conversion suits you, you must integrate the .webp images into the site (HTML) and delete the old ones.
Do you want me to do it?
